RobotCode/src/main/java/frc/robot/commands/RotateShooterMountCommand.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ShooterMountSubsystem;
+
+public class RotateShooterMountCommand extends Command {
+
+	private ShooterMountSubsystem shooterMountSubsystem;
+
+	public RotateShooterMountCommand(ShooterMountSubsystem subsytem) {
+		shooterMountSubsystem = ShooterMountSubsystem.getInstance();
+	}
+
+	private double shooterMountRotation = 0.0f; // Ranges from 0 to 180
```

Comment:
Variables should be declared above all methods, including constructors. Also, give the initial value in the constructor, rather than when declaring.